src/Infrastructure/TemplateMailerInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+use WMDE\Fundraising\Frontend\MembershipContext\Domain\Model\EmailAddress;
+
+interface TemplateMailerInterface {
```

Comment:
To make sure both the `TemplateBasedMailer` and the `LoggingMailer` expose the same method signature?